include/runtime/alloc.h

```diff
@@ -15,7 +15,7 @@ extern "C" {
 
 char youngspace_collection_id(void);
 char oldspace_collection_id(void);
-size_t youngspace_size(void);
+//size_t youngspace_size(void);
```

Comment:
let's go ahead and actually delete this; I prefer to avoid commented out code when possible.